bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/db/JdbcDerbyDAO.java

```diff
@@ -81,6 +83,13 @@ private void initDbProps() {
 
     }
 
+    @Override
+    public void initAfterFirstDbConnection() {
+        logger.debug("JDBC::initAfterFirstDbConnection: Initializing step, after db is connected.");
+        // Initialize sqlTypes, depending on DB version for example
+        // derby does not like this... dbMeta = new DbMetaData();// get DB information
+    }
+
```

Comment:
Do not understand? I have to prevent from original Function:
*// derby does not like this... dbMeta = new DbMetaData();*